libs/template/templates/default-python/template/{{.project_name}}/tests/main_test.py.tmpl

```diff
@@ -1,21 +1,15 @@
-from databricks.connect import DatabricksSession
-from pyspark.sql import SparkSession
+from databricks.connect import DatabricksSession as SparkSession
+from pytest import fixture
```

Comment:
Is pytest already available?

Reply:
It's locally available as its part of `requirements-dev.txt`